lib/generators/token_authenticate_me/models/templates/session_model.rb

```diff
@@ -3,10 +3,10 @@
 class Session < ActiveRecord::Base
   include TokenAuthenticateMe::Models::Sessionable
 
-  belongs_to :#{authenticate_model_singular_name}
+  belongs_to :<%= authenticate_model_singular_name %>
 
   def as_json(options={})
-    { #{session_model_singular_name}: super({ include: :#{authenticate_model_singular_name} }.merge(options)) }
+    { <%= session_model_singular_name %>: super({ include: :<%= authenticate_model_singular_name %> }.merge(options)) }
```

Comment:
unexpected token tCOLON<br>unexpected token tOP_ASGN<br>unterminated string meets end of file